lychee-lib/src/extract/mod.rs

```diff
@@ -8,26 +6,17 @@ mod markdown;
 mod plaintext;
 
 use markdown::extract_markdown;
-use once_cell::sync::Lazy;
 use plaintext::extract_plaintext;
 
-/// HTML elements that are deemed verbatim (i.e. preformatted).
-/// These will be excluded from link checking by default.
-static VERBATIM_ELEMENTS: Lazy<HashSet<String>> = Lazy::new(|| {
-    HashSet::from_iter([
-        "pre".into(),
-        "code".into(),
-        "textarea".into(),
-        "samp".into(),
-        "xmp".into(),
-        "plaintext".into(),
-        "listing".into(),
-    ])
-});
-
 /// Check if the given element is in the list of preformatted tags
+///
+/// preformatted tags are HTML elements that are deemed verbatim (i.e. preformatted).  These will
```

Comment:
The wording is a little weird. Reads a bit tautological: "preformatted tags are preformatted". But I get your point. 😅

Reply:
yeah that's a copypaste error. didn't read it well enough